migrations/20220117025847_email_data.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TYPE valid_status AS ENUM ('running', 'completed', 'stopped');
+CREATE TABLE blk_vrfy_job (
+    id SERIAL PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    total_records INTEGER NOT NULL,
+    job_status VALID_STATUS NOT NULL DEFAULT VALID_STATUS 'running'
```

Comment:
I'm wondering if we even need `job_status`? It's `running` if the number of items in `ema_vrfy_rec` is less than total_records